cves/CVE-2011-0722.yml

```diff
@@ -346,7 +395,20 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The vulnerability existed in two lines of code in an untested 90-line
+    function executing a complex task. This mistake comes down to a few factors:
+    - The function was too large and complex, making it difficult to understand
+    and maintain.
+    - There were no unit tests to catch edge cases such as this vulnerability.
+    - The entire decoder was implemented almost entirely by a single developer
```

Comment:
I'm not sure the single developer part is quite so bad, but I would focus more on the lack of review if that's the case.

Reply:
I think they both have equal weight - in either case, the lack of additional eyes meant nobody would catch the little things that the original developer could easily overlook.